src/DataWrangling/ECCO2.jl

```diff
@@ -1,111 +1,173 @@
 )
 
-ecco2_file_names = Dict(
-    :temperature             => "ecco2_temperature_19920102.nc",
-    :salinity                => "ecco2_salinity_19920102.nc",
-    :effective_ice_thickness => "ecco2_effective_ice_thickness_19920102.nc",
+urls_19921001 = Dict(
+    :temperature           => "https://www.dropbox.com/scl/fi/169f3981460uhk9h69k0f/THETA.1440x720x50.19921001.nc?rlkey=mgal3xt0qy2c59y395ybio11v",
+    :salinity              => "https://www.dropbox.com/scl/fi/f9zfm34vqz732jrrhjrg3/SALT.1440x720x50.19921001.nc?rlkey=y5dv0s41gb6f9guvu0iorw28p",
+    :sea_ice_thickness     => "https://www.dropbox.com/scl/fi/mtmziurepom8kpjn82d07/SIheff.1440x720.19921001.nc?rlkey=9uhuxg2n9iw6894afj4t53drv",
+    :sea_ice_area_fraction => "https://www.dropbox.com/scl/fi/ntflhyrmsnit9vco402co/SIarea.1440x720.19921001.nc?rlkey=eakzc788btql1q6ndj9l8cr2q",
+    #:u_velocity            => "https://www.dropbox.com/scl/fi/e6s9c013r2ddift4f8ugi/UVEL.1440x720x50.19921001.nc?rlkey=fpd7mv1zv3fkmyg8w11b94sbp&dl=0",
+    :u_velocity            => "https://www.dropbox.com/scl/fi/e6s9c013r2ddift4f8ugi/UVEL.1440x720x50.19921001.nc?rlkey=fpd7mv1zv3fkmyg8w11b94sbp&dl=0",
+    :v_velocity            => "https://www.dropbox.com/scl/fi/nxuohvhvdu0ig552osf1d/VVEL.1440x720x50.19921001.nc?rlkey=vz4ttp3myxhertdxvt1lyjp1d",
 )
 
-# Downloaded from https://ecco.jpl.nasa.gov/drive/files/ECCO2/cube92_latlon_quart_90S90N
-
-ecco2_urls = Dict(
-    :temperature => "https://www.dropbox.com/scl/fi/01h96yo2fhnnvt2zkmu0d/" *
-                    "THETA.1440x720x50.19920102.nc?rlkey=ycso2v09gc6v2qb5j0lff0tjs&dl=0",
+filenames = Dict(
+    "1992-01-02" => filenames_19920102,
+    "1992-10-01" => filenames_19921001,
+)
 
-    :salinity => "https://www.dropbox.com/scl/fi/t068we10j5skphd461zg8/" *
-                 "SALT.1440x720x50.19920102.nc?rlkey=r5each0ytdtzh5icedvzpe7bw&dl=0",
+urls = Dict(
+    "1992-01-02" => urls_19920102,
+    "1992-10-01" => urls_19921001,
+)
 
-    :effective_ice_thickness => "https://www.dropbox.com/scl/fi/x0v9gjrfebwsef4tv1dvn/" *
-                                "SIheff.1440x720.19920102.nc?rlkey=2uel3jtzbsplr28ejcnx3u6am&dl=0"
+shortnames = Dict(
+    :temperature           => "THETA",
+    :salinity              => "SALT",
+    :sea_ice_thickness     => "SIheff",
+    :sea_ice_area_fraction => "SIarea",
+    :u_velocity            => "UVEL",
+    :v_velocity            => "VVEL",
 )
 
-function construct_vertical_interfaces(ds, depth_name)
-    # Construct vertical coordinate
-    depth = ds[depth_name][:]
-    zc = -reverse(depth)
 
-    # Interface depths from cell center depths
-    zf = (zc[1:end-1] .+ zc[2:end]) ./ 2
-    push!(zf, 0)
-    
-    Δz = zc[2] - zc[1]
-    pushfirst!(zf, zf[1] - Δz)
 
-    return zf
-end
+surface_variable(variable_name) = variable_name == :sea_ice_thickness
 
-function ecco2_field(variable_name;
+function ecco2_field(variable_name, date=Date(1992, 01, 02);
                      architecture = CPU(),
```

Comment:
if `ecco2_field` is architecture-depended, then users of HPCs that don't have Internet access on CPU/GPU compute nodes how are they supposed to download it?

for example, on Australia's gadi only the login node has internet access and that's a CPU architecture...

Reply:
the downloading is not architecture dependent

Reply:
you can build a wrapper for this step:

https://github.com/CliMA/ClimaOcean.jl/blob/9125cb5e59334c3705879424952fb0582b98ed3b/src/DataWrangling/ECCO2.jl#L69

just need to design the API